app/models/child_object.rb

```diff
@@ -128,7 +128,7 @@ def pyramidal_tiff
   end
 
   def thumbnail_url
-    "#{IiifPresentationV3.new(parent_object).image_service_url(oid)}/full/!200,200/0/default.jpg"
+    "#{IiifPresentationV3.new(parent_object).image_service_url(oid)}/full/!1200x630/0/default.jpg"
```

Comment:
Think these may need to be commas not xs

Reply:
I fixed it.  Copy, cut and dump effect